README.md

```diff
@@ -2,7 +2,191 @@
+This is a collection of databases that specify mathematical axioms and
+rigorous formal proofs of various mathematical theorems
+derived from those axioms.
+This collection is specified and derived using the Metamath system.
+A "database" in Metamath parlance is a collection of one or more text files.
 
-File "set.mm" uses classical logic and ZFC set theory.
-File "iset.mm" uses intuitionistic logic.
+This collection is being actively improved by an international community.
+Please join us!
+
+The two largest databases in this collection are "set.mm" and "iset.mm".
+The "set.mm" database is one of the largest
+collections of formally-verified mathematics in the world (e.g., it
+has completed many challenge theorems in the
+[Formalizing-100 Theorems](https://www.cs.ru.nl/~freek/100/) challenge list).
+It is also one of the most rigorously verified collection of
+formally-verified mathematics in the world; it's verified by
+5 different verifiers written by 5 different people
+in 5 different programming languages.
+
+## What is Metamath?
+
+Metamath is a computer language and associated computer program for
+archiving, verifying, and studying mathematical proofs.
+
+Unlike some other systems, Metamath does not build a particular set
+of axioms into the system. Instead, the Matamath language is simple and robust,
+with an almost total absence of hard-wired syntax.
+In Metamath you express the axioms *and* the theorems in a database.
+To prove a theorem, every proof step *must* be proven using an axiom or
+a previously-proven theorem; as a result, nothing is hidden.
+We believe Metamath provides about the simplest possible framework that
+allows essentially of mathematics to be expressed with absolute rigor.
+
+For more information see
+the [Metamath Home Page](http://us.metamath.org/), the
+[Metamath Proof Explorer Home Page](http://us.metamath.org/mpeuni/mmset.html),
+the [Metamath book](http://us.metamath.org#book), or the
+[video "Metamath Proof Explorer: A Modern Principia Mathematica](https://www.youtube.com/watch?v=8WH4Rd4UKGE).
+
+## What databases are included in this collection?
+
+This is a collection of many databases, but since "set.mm" is by far
+the most actively maintained, you may want to look at it first even if you
+plan to eventually work on something else.
+[The video "Metamath Proof Explorer (set.mm) contributions visualized with Gource through 2019-10-04"]( https://www.youtube.com/watch?v=XC1g8FmFcUU&list=PL1jSu6GGefBm7RBP0Id2Sa9uyVuyhioAC&index=4) shows set.mm's growth over time.
+
+The databases included, in (approximate) decreasing size, are:
+
+* "set.mm" aka Metamath Proof Explorer (MPE) -
+  uses classical logic and
+  Zermelo–Fraenkel set theory with the axiom of choice (ZFC).
+* "iset.mm" aka "Intuitionistic Logic Explorer" -
+  uses intuitionistic logic instead of classical logic,
+  e.g., it does not presume that the law of the implied middle is true
+  in all cases.
```

Comment:
You may want to note that iset.mm is intuitionistic *set theory*, because there are several other varieties of intuitionistic mathematics. Specifically, I think iset.mm uses IZF (which is strictly stronger than CZF, another version of intuisionitic set theory).

Reply:
There is some CZF starting with the section "Bounded formulas" just above http://us.metamath.org/ileuni/ax-bd0.html . Most of iset.mm is IZF, though.

And of course it is true that this is all set theory, not type theory or another style of foundation.

How deep to get into this in the README, I'm not quite as clear, but I'm all for explaining these distinctions more, here and/or elsewhere.

Reply:
I don't think we need to get into IZF vs. CZF here. If someone's interested, they'll jump to that.

Revised summary;

* "iset.mm" aka "Intuitionistic Logic Explorer" -  uses intuitionistic set theory.  In particular, it does not presume that the law of excluded middle is  necessarily true in all cases.